libpod/networking_pasta.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// networking_pasta.go - Start pasta(1) to provide connectivity to the container
+//
+// Copyright (c) 2022 Red Hat GmbH
+// Author: Stefano Brivio <sbrivio@redhat.com>
+//
+// +build linux
+
+package libpod
+
+import (
+	"os/exec"
+	"fmt"
+	"strings"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/pkg/errors"
```

Comment:
deprecated

Reply:
this should just use `errors`, instead of `errors.Wrap` use `fmt.Errorf("msg: %w",err)`

Reply:
Oh, sorry, I wasn't aware. I dropped the `import` and changed to `fmt.Errorf()` -- I kept the `err.(*exec.ExitError).Stderr` part, though, because pasta(1) prints informative output to `stderr` as to the reason why it couldn't start.